epam_tr/src/main/java/com/epam/tr/security/request/RequestProcessor.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.tr.security.request;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+
+import static com.epam.tr.security.secret.JWTSecretHolder.JWT_SECRET;
+import static org.springframework.util.StringUtils.hasText;
+
+@Component
+public class RequestProcessor {
+
+    private static final String AUTHORIZATION = "Authorization";
+    private static final String BEARER = "Bearer ";
+
+    public String getTokenFromRequest(HttpServletRequest request) {
+        String bearer = request.getHeader(AUTHORIZATION);
+        if (hasText(bearer) && bearer.startsWith(BEARER)) {
+            return bearer.substring(7);
```

Comment:
Should be a better way to check if token is in a request.
If I remebmer correctly spring has api to do so.